packages/datacommons-mcp/datacommons_mcp/server.py

```diff
@@ -449,25 +371,55 @@ async def search_topics_and_variables(
     candidates and filter them based on the user's query and context to surface the most
```

Comment:
followup: consider something like:

```
This tool returns candidate indicators that match your query. Use your discretion to determine the extent to which each candidate relates to your query given your specific usecase.
```
which is more generic about how to understand the response rather than instruct how to use it.